Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml

```diff
@@ -2108,6 +2108,40 @@ inputs:
   required: false
   description: "Possible values: True/False.\nWhether to provide user verification for blocking IPs. \n\nFalse - No prompt will be displayed to the user.\nTrue - The server will ask the user for blocking verification and will display the blocking list."
   playbookInputQuery:
+inputSections:
+- inputs:
+  - LinkSimilarIncidents
+  - CriticalUsernames
+  - CriticalHostnames
+  - CriticalADGroups
+  name: Incident Management
```

Comment:
I dont see a point to have incident management and incident data sections. 
Lets combine it, or lets talk about it.

Reply:
done